### ISSUE TITLE
Upgrades behind the creature and cannot be viewed

### ISSUE THREAD

**Issue by AussieJedi** [jeremylarner/keyteki#221]
Hi I lost a game because I couldnt see that a jammer pack was costing my keys to be +2. It was behind a creature that was given the upgrade, but because he had an artifact there, it really covered up the line of sight. when I lost the game, he taunted me like he knew what he was doing. It is pretty shady for a player to do that. 

Is there some way that we can view an upgrade on a character?

**Comment by 4ugeistr** [comment]
Yeah, that would be nice. Got confused about that too a couple of times. Even if you see the upgrade outline, there's no way to see what kind of upgrade is that. You have to look through the log to see what kind of card has been played.

A horizontal offset could help, since it would add visibility and a little area to hover over a card and see its details.
I'd also put upgrades above artifacts, since that would still leave the artifact visible and would not hide the artifact entirely.

**Comment by 4ugeistr** [comment]
Duplicate of [Issue #211](https://github.com/jeremylarner/keyteki/issues/211)
